app/controllers/moderators_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ModeratorsController < ApplicationController
+  before_action :authenticate_person!
+  after_action :verify_authorized
+
+  def update
+    authorize(community, policy_class: ModeratorPolicy)
```

Comment:
Not Blocking: You want your policy classes to reflect the object being mutated; not the person doing the mutation. 

In this particular method, we are updating a Membership, not a community. As such, we want to have the policy and scopes tied to the Membership.

That said, a ModeratorPolicy "feels nice" but it's not like we're changing a _moderator_. We're changing a Membership to toggle the moderator status for a person within a particular community.

I know it's kind of a weird distinction, and it may not even matter really. However, when we're working with highly opinionated frameworks (like Rails) it's generally better to lean into the grain; and a keyword arg for the policy_class feels like it's "against" the grain.